### PR TITLE
Update zxcvbn Welsh language pack

### PR DESCRIPTION
**Pull request by deangray82**
Part of kiddivouchers/daisy#7404

### CODE REVIEW

**Review by la5942 — APPROVED** [review]
LGTM

**Comment by cs278** [comment]
Does this need to bump the version number @Jamesking56 ?

**Comment by deangray82** [comment]
> Does this need to bump the version number

I've bumped the version number. Should this use tags? It looks like it uses git (master branch) on daisy (https://github.com/kiddivouchers/daisy/blob/develop/package.json#L5-L5).

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@widerplan/zxcvbn-language-cy",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (cs278) [inline package.json]: This is a breaking change

Comment (deangray82) [reply package.json]: Updated, thanks.

**Review by Jamesking56 — APPROVED** [review]

**Comment by cs278** [comment]
> Should this use tags? It looks like it uses git (master branch) on daisy

@Jamesking56 might know this, I'm not sure.

**Pull request merged**